bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1478,17 +1478,18 @@ namespace JsRTApiTest
     {
         JsValueRef result = JS_INVALID_REFERENCE, callbackState = JS_INVALID_REFERENCE, exception = JS_INVALID_REFERENCE;
         JsValueType cbStateType = JsUndefined;
+        const wchar_t *script = _u("new Promise((res, rej) => res()).then(() => 1)");
```

Comment:
Oh, hmm, it looks like maybe these tests are currently Windows only?  In that case I guess it doesn't matter, but we should probably get them running cross-platform.